### ISSUE TITLE
ProtocolTestManager.exe UI is not responsive

### ISSUE THREAD

**Issue by jtmoon79** [microsoft/WindowsProtocolTestSuites#186]
**tl;dr** the `ProtocolTestManager.exe` does not respond to user input (mouse or keyboard)

## Reproduction

1. Using 84fee9e5d3f0a7b00945483bf43933022f4bda91
2. I installed the prerequisites recommended in https://github.com/microsoft/WindowsProtocolTestSuites/blob/84fee9e5d3f0a7b00945483bf43933022f4bda91/README.md
3. I built the `FileServer` and `MS-SMB` Test Suites, and the `ProtocolTestManager`
```
cd WindowsProtocolTestSuites/TestSuites/FileServer/src
.\build.cmd

(build succeeded)

cd WindowsProtocolTestSuites/TestSuites/MS-SMB/src
.\build.cmd

(build succeeded)

cd WindowsProtocolTestSuites/ProtocolTestManager
.\build

(build succeeded)
```
4. try directly running `WindowsProtocolTestSuites/ProtocolTestManager/Bin/ProtocolTestManager.exe`
  a. double-click `WindowsProtocolTestSuites/ProtocolTestManager/Bin/ProtocolTestManager.exe`
  b. Try to select a setting ***BUG:*** does not respond to keyboard or mouse input (see picture)
5. try `WindowsProtocolTestSuites/ProtocolTestManager/installer/ProtocolTestManager.msi`
  a. install `WindowsProtocolTestSuites/ProtocolTestManager/installer/ProtocolTestManager.msi`
  b. from Start Menu, run _Protocol Test Manager_
  c. Try to select a setting ***BUG:*** does not respond to keyboard or mouse input (see picture)

### Details

[Windows Protocol Test Manager Issue#186](https://imgur.com/U4dqnnn)

Running on Windows 10 Pro, Powershell 5.1.

**Comment by huiren-jiang** [comment]
Hi James,

Have you installed any test suite? Protocol Test Manager lists all supported test suites on welcome page and "Configuration Wizard" will appear for each if the corresponding test suite is installed.

Best Regards,

Windows Protocol Test Suites Development Team

**Comment by jtmoon79** [comment]
> Have you installed any test suite? 

How do I install a test suite?

**Comment by jtmoon79** [comment]
> Have you installed any test suite? 

<strike>How do I install a test suite?</strike>

Ah, I didn't realize I had to also install each Test Suite .msi file.  e.g. `WindowsProtocolTestSuites/drop/TestSuites/FileServer/deploy/FileServer-TestSuite-ServerEP.msi`.  

Maybe this should be added to the _Run_ section or _Build_ section of the `README.md`?

**Comment by huiren-jiang** [comment]
Hi James,

Thanks for your feedback. After you created this issue, we also realized this confusing part and planned to make improvement to readme.md and PTM.

Best Regards,

Windows Protocol Test Suites Development Team
